src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
```

Comment:
despite the fact that fetch works without installing additional packages, I would prefer to use axios.
Axios automatically transforms the data returned from the server, but with fetch() you have to call the response.json method to parse the data to a JavaScript object.
Axios is less vulnerable because it provides client-side CSRF protection.
Axios has a timeout that determines the time limit after which the request is canceled, and also handles errors more easily.